.gitignore

```
@@ -0,0 +1 @@
+node_modules
```

```suggestion
node_modules
.env
```
Add your .env to this file and leave a .env.example file in your project for the next dev.